gems/manageiq-providers-amazon/spec/models/manageiq/providers/amazon/cloud_manager/provision_workflow_spec.rb

```diff
@@ -110,9 +110,8 @@
   context "with applied tags" do
     before do
       FactoryGirl.create(:classification_cost_center_with_tags)
-      admin.current_group.set_managed_filters([['/managed/cc/001']])
-      admin.current_group.set_belongsto_filters([])
-      admin.current_group.save
+      admin.current_group.entitlement = Entitlement.create!(:filters => {'managed'   => [['/managed/cc/001']],
```

Comment:
Sidenote; Doing this (shoving this hash literal everywhere) all over is definitely not ideal, but all of this will have to change at least twice with filters likely changing format (for merging) and with the introduction of multiple entitlements - so putting forth too much effort thinking of a more efficient way is moot.